index.html

```diff
@@ -0,0 +1,269 @@
+                <li><a href=""><img class="icons" src="images/Linkedin icon@2x.png" alt=""></a></li>
+                <li><a href=""><img class="icons" src="images/angel list@2x.png" alt=""></a></li>
+                <li><a href=""><img class="icons" src="images/twitter@2x.png" alt=""></a></li>
+                <li><a href=""><img class="icons" src="images/medium@2x.png" alt=""></a></li>
+
+            </ul>
+
+            <div class="main-info">
+                <h1 class="info-header">Hey There.<br>I'm Ian.</h1>
+                <p class="info-title">I'm a Software Developer</p>
+                <p class="info-details">I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hesitate to contact me.</p>
+
+            </div>
+
+        </main>
+   </header>
+
+   <!-- Recent Work Section-->
+   <section class="work-container">
+
+        <div class="project-title">
+            <div class="header-title">
+                <h1>My Recent Works</h1>
+            </div>
+            <div class="line-container">
+                <hr>
+             </div>
+        </div>
+
+
+        <div class="main-project">
+
+            <div class="main-img">
+                <img src="images/Img Placeholder.png" alt="">
+            </div>
+
+            <div class="main-content">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul>
+                    <li>CSS</li>
+                    <li>HTML</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button>See project</button>
+            </div>
+
+        </div>
+
+        <div class="project-container">
+
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h1>Profesional Art Printing Data</h1>
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                  <ul>
+                      <li>HTML</li>
+                      <li>Bootstrap</li>
+                      <li>Ruby</li>
+                  </ul>
+              </div>
+                <button>See project</button>
+            </div>
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h1>Profesional Art Printing Data</h1>
```

Comment:
- Kindly you will have one `h1` element per page, so use other heading elements for this for better semantic arrangement 👍